src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.model;
+
+class Car {
```

Comment:
메서드 배치 순서가 default -> private 순서로 생각하고 작성하였고, default는 같은 패키지 내 다른 클래스에서 쓰이지만 private은 내부에서만 쓰이니 default를 위로 하는 게 보기 쉽다고 생각하였습니다. 그런데 코니가 남겨주신 글을 보고 검색해보니 private -> default 순서가 컨벤션이었군요.. 

> Checks that the order of modifiers conforms to the suggestions in the [Java Language specification, § 8.1.1, 8.3.1, 8.4.3](https://docs.oracle.com/javase/specs/jls/se16/preview/specs/sealed-classes-jls.html) and [9.4](https://docs.oracle.com/javase/specs/jls/se11/html/jls-9.html). The correct order is:
> 
> public
> protected
> private
> abstract
> default
> static
> sealed
> non-sealed
> final
> transient
> volatile
> synchronized
> native
> strictfp

Reply:
처음에는 넓은 scope 순서로 배치하는 것이 위에 다른 클래스에서 사용하는 메소드가 위치하고, 아래 클래스 내부에서만 이용하는 메소드가 위치하여 좋다고 생각하였습니다. 코니가 답변주신 질문을 보고 일반적인 컨벤션이 있나 찾아보았는데, 

> These methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.
(Code Conventions for the java(Oracle)

> The order you choose for the members and initializers of your class can have a great effect on learnability. However, there's no single correct recipe for how to do it; different classes may order their contents in different ways.
> What is important is that each class uses some logical order, which its maintainer could explain if asked. For example, new methods are not just habitually added to the end of the class, as that would yield "chronological by date added" ordering, which is not a logical ordering.
> (Google Java Style Guide)
글을 보고 scope 보다는 메소드의 기능별로 응집도 있도록 논리적으로 가독성 있게 배치하는 것이 옳은 배치라고 생각하였습니다. 2단계부터 적용해보겠습니다.